blog/apps.py

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
```suggestion
```

Reply:
For educational purpose used rebase -i to fix this issue)

Reply:
sure. it's often useful to accept a lot of small patches first and then do interactive rebase and apply them as fixups in different places. this can be done after some rounds of review.